src/app/(post-login)/submission/program/[shortName]/clinical-submission/data.tsx

```diff
@@ -39,10 +39,12 @@ const CLINICAL_FILE_ORDER = [
 
 // parse out nulls, undefined, provide sensible defaults so type checking isnt a scattered nightmare
 export const parseGQLResp: (v: any) => ClinicalSubmission = (gqlData) => {
```

Comment:
don't we have generated types for gql? if we change the function signature we should be able to use them.